python/delta_sharing/delta_sharing.py

```diff
@@ -104,6 +104,7 @@ def load_as_pandas(
     version: Optional[int] = None,
     timestamp: Optional[str] = None,
     jsonPredicateHints: Optional[str] = None,
+    useKernel: Optional[bool] = None,
```

Comment:
@linzhou-db I was not sure whether to name this as use_kernel or useKernel. the rest of variables in python are not in camelCase but jsonPredicateHints is. Should I make both of them use the normal python_underscore_naming_convention?

Reply:
jsonPredicataHints is considered a full string. so let's use the underscore naming. 

though, I'm thinking of calling it `use_delta_format`